send the right type expected instead of a string

greatalf opened the pull request:
<!--

If your pull request fixes a BUG, use the oldest maintained branch that contains
the bug (see https://symfony.com/roadmap for the list of maintained branches).

If your pull request documents a NEW FEATURE, use the same Symfony branch where
the feature was introduced (and `master` for features of unreleased versions).

-->

fmata committed:
Update dotenv.rst

kbond committed:
document ServiceSubscriberTrait

yceruto committed:
Document setDeprecated method from OptionsResolver component

jderusse committed:
Add Lock's PdoStore documentation

Commit:
[HttpFoundation] Add info for getAcceptableFormats() method

javiereguiluz committed:
minor #9809 [DependencyInjection] Document ServiceSubscriberTrait (kbond)

This PR was merged into the master branch.

Discussion
----------

[DependencyInjection] Document ServiceSubscriberTrait

Documentation for https://github.com/symfony/symfony/pull/27077

Commits
-------

b0ac3a4f0 document ServiceSubscriberTrait

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Better explain render() and render_esi() functions
  Updated the embed controllers article
  Use <button> instead of <a> for JavaScript interactive buttons

xabbuh committed:
Merge branch '4.1'

* 4.1:
  add missing label

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Update http_foundation.rst
  Use matching closing tag

colinodell committed:
[Form] Ability to clear form errors

OskarStark committed:
use is_granted() instead of deprecated has_role()

Deprecated in https://github.com/symfony/symfony/pull/27305

colinodell committed:
Clarify that clearing errors makes the form valid

javiereguiluz committed:
minor #9919 use is_granted() instead of deprecated has_role() (OskarStark)

This PR was merged into the master branch.

Discussion
----------

use is_granted() instead of deprecated has_role()

using `has_role()` was deprecated in https://github.com/symfony/symfony/pull/27305 since `4.2` by @nicolas-grekas

Commits
-------

5e6031b34 use is_granted() instead of deprecated has_role()

javiereguiluz committed:
minor #9898 [HttpFoundation] Add info for getAcceptableFormats() method (AndreiIgna)

This PR was squashed before being merged into the master branch (closes #9898).

Discussion
----------

[HttpFoundation] Add info for getAcceptableFormats() method

Adds info for `Request::getAcceptableFormats()`

Should it add more info on docs page? Ex: `Request::getAcceptableContentTypes` returns the accepted content types which can be `text/html`, `application/json`, etc.
`Request::getAcceptableFormats` processes that and returns the formats based on those content types `html`, `json`

ref https://github.com/symfony/symfony/pull/26486

Commits
-------

65d100eb5 [HttpFoundation] Add info for getAcceptableFormats() method

javiereguiluz committed:
minor #9859 [OptionsResolver] Document setDeprecated method (yceruto)

This PR was merged into the master branch.

Discussion
----------

[OptionsResolver] Document setDeprecated method

Documenting feature https://github.com/symfony/symfony/pull/27277.

Commits
-------

afa5e54f8 Document setDeprecated method from OptionsResolver component

javiereguiluz committed:
Added the versionadded directive

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Improved the front controller of the HttpCache
  Better docs about overriding third-party translations
  event constant mistake
  Updated the default values of some Cache config options
  add legacy labels
  Update Best Practices about service config format
  Update simple-example.rst to use const for jQuery
  Update shared-entry.rst to use Bootstrap 4's SCSS
  Removed an extra blank line
  Reword and added a note
  Link to proxy configuration docs from the routing scheme page
  Fixed typos
  Fixed security expression testing user
  Update messenger.rst
  Fix a configuration block in the Doctrine Associations page
  Fix numbering
  Mentioned Debian explicitly

weaverryan committed:
Merge branch '4.1'

* 4.1:
  Removed a short article about overriding bundle templates
  Fix configuration-block not working
  Change form.xxx_bind to form.xxx_submit

weaverryan committed:
Merge branch '4.1'

* 4.1:
  Removed a short article about overriding bundle templates
  Improved the front controller of the HttpCache
  Better docs about overriding third-party translations
  Change form.xxx_bind to form.xxx_submit
  add legacy labels
  Update Best Practices about service config format
  Fix a configuration block in the Doctrine Associations page
  Fix numbering
  add missing label
  Minor typo in csrf.rst
  Update setup.rst

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Updated associations.rst
  Formatting: Apply PSR-2 formatting to code example
  Update conventions.rst
  Added a missing versionadded directive for 307/308 redirects
  Wrap long lines
  State Maker dependency for generating controllers
  Small grammar update in HttpFoundation

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Fixed more occurrences of the wrong handler name
  bugfixes should be placed for the 2.8 branch

javiereguiluz committed:
Wrap long lines and add the versionadded directive

javiereguiluz committed:
minor #9916 [Form] Ability to clear form errors (colinodell, javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

[Form] Ability to clear form errors

Documenting the new feature proposed in https://github.com/symfony/symfony/pull/27580

Commits
-------

f07f53611 Wrap long lines and add the versionadded directive
7a4465d4d Clarify that clearing errors makes the form valid
d4143f15a [Form] Ability to clear form errors

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Removed the full config dumps from config reference articles
  Mention that the component implements both patterns
  Update event_dispatcher.rst
  Observer instead of Mediator pattern
  Recommend to use real env vars in production
  Update dotenv usage

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Fix some namespaces
  Minor reword and fixes
  Add section about anonymous services
  Add iterable to the Type constraint
  Added a note about hidden questions, windows and tests

dmaicher committed:
Update event_listeners_subscribers.rst

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Use the ReadTheDocs theme as the default Sphinx theme
  Added the missing versionadded directive
  [ExpressionLanguage] Add missing argument in example code
  Explained some issues about command constructors and configure
  Install component for dev environment
  Fix not precise enough sentence
  Add new serializer empty_data feature doc

javiereguiluz committed:
Reworded and added the versionadded directive

javiereguiluz committed:
Update event_listeners_subscribers.rst

javiereguiluz committed:
Explain that lazy listeners are mandatory, not default

javiereguiluz committed:
minor #9973 [DoctrineBridge] update doctrine event listeners doc for Symfony 4.2 change (dmaicher, javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

[DoctrineBridge] update doctrine event listeners doc for Symfony 4.2 change

See https://github.com/symfony/symfony/pull/27675.

Doctrine entity listeners are always lazily instantiated as of Symfony 4.2.

Commits
-------

4409fd65b Explain that lazy listeners are mandatory, not default
b56ef9eaf Update event_listeners_subscribers.rst
d8bb8492b Reworded and added the versionadded directive
601450e25 Update event_listeners_subscribers.rst

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  [testing] tell about self::$container
  Fix typo in the Twig article
  Removed some whitespace
  Reworded the tip contents
  Add Tip to using  MailHog during developement
  Added a small note about dynamic role hierarchies
  Added a note about command aliases and services

javiereguiluz committed:
Added some missing link references

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Update bootstrap.rst
  Fix PHPUnit versions

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Fixed a RST syntax issue

nicolas-grekas committed:
[Cache] add Redis options compression and tcp_keepalive

Commit:
json-login-ldap

nicolas-grekas committed:
Tell about ProcessorInterface in logging/processors

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Fixed more Twig class namespaces
  Fix Twig namespaces and update doc for twig runtime extension
  Popper js is required
  ChoiceValidator : remove 'strict' option
  Proposed a reword abut the "lazy" explanation
  Tell about ServiceLocatorTagPass::register()
  [Cache] Add note about lazy=true option

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  More fixes about Twig classes namespaces

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Improved the configuration references
  Fixed inconsistency differently
  Update entity_provider.rst
  Update scheme.rst

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Fixed a minor sorting error

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Fixed another issue cause by options reorganization

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Fixed more sorting issues in config options
  Removed the full config dump of FrameworkBundle

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Fixed a nested option syntax issue

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Fixed some nested options issues

bobvandevijver committed:
Added key env processor to docs

javiereguiluz committed:
Reworded and added other config formats

javiereguiluz committed:
minor #10009 Added key env processor to docs (bobvandevijver, javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

Added key env processor to docs

Not much more to say, this is documenting the feature added in symfony/symfony#27157.

Fixes #9734.

Commits
-------

9c91cf1cf Reworded and added other config formats
1d72bc25d Added key env processor to docs

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Use AbstractController in the main controller article
  Minor rewords
  Fix broken link to `vue-loader` documentation.
  Simplify Apache config with FallbackResource
  Made the code of a validation example more robust
  Tell about controllers in doc about service subscribers

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Removed the code of the PHP templates

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Removed more PHP template examples
  [Messenger] Minor wording tweak regarding factories

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Removed the PHP templates from Symfony 4.0 docs

javiereguiluz committed:
Minor reword

javiereguiluz committed:
minor #9996 Tell about ProcessorInterface in logging/processors (nicolas-grekas, javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

Tell about ProcessorInterface in logging/processors

Doc for https://github.com/symfony/symfony/pull/27801

Commits
-------

ad2bad914 Minor reword
0c9c90fdf Tell about ProcessorInterface in logging/processors

javiereguiluz committed:
minor #9985 [Cache] add Redis options compression and tcp_keepalive (nicolas-grekas)

This PR was merged into the master branch.

Discussion
----------

[Cache] add Redis options compression and tcp_keepalive

Since https://github.com/symfony/symfony/pull/27646

Commits
-------

266020c43 [Cache] add Redis options compression and tcp_keepalive

javiereguiluz committed:
Added the missing link

GaneshChandrasekaran-zz committed:
Add docs for Zookeeper Data Store.

javiereguiluz committed:
minor #9987 json-login-ldap (Rudy Onfroy)

This PR was merged into the master branch.

Discussion
----------

json-login-ldap

https://github.com/symfony/symfony/pull/27650

Commits
-------

de28118eb json-login-ldap

javiereguiluz committed:
Added the missing "versionadded" directive

fmata committed:
Update dotenv.rst

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  [#9803] fix minor typos
  [#9836] fix a minor typo
  Fix the page title for the mbstring polyfill
  update Twig documentation links
  Update licence hint for JMS serializer
  Update external_resources.rst
  Add traits to BC policy
  Mention the DoctrineTestBundle for tests that interact with the database
  Added the missing versionadded directive
  Remove duplicate `use` keyword
  Add missing RuntimeExtensionInterface
  Removed the "Learn More" section until Messenger adds more articles
  Added a note about using the component in Symfony apps
  Added a learn more section
  Document Twig test "rootform"

xabbuh committed:
Merge branch '4.1'

* 4.1:
  [#10036] (hopefully) final indentation fix
  [#10036] fix indentation
  fix code block indentation

jfredon committed:
[Finder] Add documentation and code on array as method parameter

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Replace more occurrences of Controller by AbstractController
  Fixed a missing AbstractController occurrence
  Update page_creation.rst
  the product category must be nullable
  Added the versionadded directive
  remove reference to Assetic
  [Encore] Fix vue-loader installation
  Add section about anonymous services in YAML
  update the serializer cache description
  render versionadded directive outside the code block
  remove remaining AppBundle references
  Update routing.rst

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Remove redundant example configuration value
  Moved PHP-related code from Forms to the PHP templating page
  method name is 'index' not 'indexAction'
  add XML and PHP code blocks

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Remove some unneeded changes
  Don't use env vars in all config examples
  add more trailing comma on last array element #9493
  add trailing comma on last array element #9493
  codestyle
  add documentation for external parameters and pdo host parameter

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Update service_container.rst
  hide Assetic pages from TOC
  Renaming variable
  [Validation]  Added internal link
  Reword the header regarding the envelope
  Add message envelope in the documentation

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Improved the section about overriding bundle services
  Link to the env vars article from the Best Practices
  Added a caution note about race conditions in EntityType
  Minor reword of deployment article when using Flex
  Mention how to inject a specific Monolog service

javiereguiluz committed:
Added docs for clickLink() and submitForm()

dunglas committed:
[Security] Remember me: samesite cookie

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Codefence ClassMetadataFactory
  Document the required ClassMetadataFactory
  Update framework.rst
  Fixed the preferred_choices option for Enttiy type
  Mentioned the 767 bytes index limit and its solution
  Refactor a security code example to make it easier to understand
  Remove deleted RoleInterface
  Fix How to dynamically Generate Forms Based on user Data
  Fixed the default value of php_errors.log option

javiereguiluz committed:
minor #10077 [Security] Remember me: samesite cookie (dunglas)

This PR was merged into the master branch.

Discussion
----------

[Security] Remember me: samesite cookie

https://github.com/symfony/symfony/pull/27976

Commits
-------

eefa1829c [Security] Remember me: samesite cookie

javiereguiluz committed:
minor #10071 Added docs for clickLink() and submitForm() (javiereguiluz)

This PR was squashed before being merged into the master branch (closes #10071).

Discussion
----------

Added docs for clickLink() and submitForm()

This documents https://github.com/symfony/symfony/pull/27807

Commits
-------

0668ccced Added docs for clickLink() and submitForm()

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Updated the default value of property_info.enabled option
  Fix a "duplicated reference" issue
  server_version should always be a string
  Document missing property_info node in framework configuration
  Remove typo in example constructor
  make the care team list actually in alphabetic order
  Add missing colon to reserved chars

chalasr committed:
Fix typo in remember_me chapter

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Fix the YAML syntax highlighting

javiereguiluz committed:
minor #10083 Fix typo in remember_me chapter (chalasr)

This PR was merged into the master branch.

Discussion
----------

Fix typo in remember_me chapter

Commits
-------

90c2b72bd Fix typo in remember_me chapter

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Image to console
  Transport not mandatory
  Show console table examples in a console terminal
  Merged the output of another command with its command line
  Showed the output of the command in the same CLI window
  Fixed some wrong examples about service config
  Replace entry/section by "article"
  Remove occurrences of "performant"

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Updated the Symfony Flex server
  Updated some config for Symfony 4 practices
  Documented the findByCodes() validation method
  Fixed the type of the twig.cache config option
  Update the argument value resolver article
  Documented the catchExceptions() method
  Documented the enableExceptionOnInvalidIndex() method
  Added the missing attr option in TextType

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Fixed some config file paths for Flex and Symfony 4

Commit:
[Routing] Add locale fallback to internationalized routes

nicolas-grekas committed:
[Cache] tell about PDO configuration and table autocreation

javiereguiluz committed:
minor #10111 [Cache] tell about PDO configuration and table autocreation (nicolas-grekas)

This PR was merged into the master branch.

Discussion
----------

[Cache] tell about PDO configuration and table autocreation

Doc for https://github.com/symfony/symfony/pull/27694

Commits
-------

187300adb [Cache] tell about PDO configuration and table autocreation

javiereguiluz committed:
Fixed a minor RST issue

yceruto committed:
Document file_relative Twig function

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Different way for updating flex projects
  Update sessions.rst
  Removed some repeated content and minor rewords
  doc(testing/http_authentication.rst);
  Update locale_sticky_session.rst
  Update form_collections.rst
  Add necessary call to Form::isSubmitted and remove boilerplate
  Undefined variable $userName in example
  [Cache] minor tweaks
  Fixed the code of the custom password authenticator example
  Documented PHPUnit data providers
  [Routing] do not install SensioFrameworkExtraBundle

javiereguiluz committed:
Reword and added the versionadded directive

javiereguiluz committed:
Final reword

javiereguiluz committed:
minor #10115 Add file_relative function to the Twig reference (yceruto, javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

Add file_relative function to the Twig reference

Documenting https://github.com/symfony/symfony/pull/27978/files#diff-f1a4cceb03e213bc53ab3f0ed53b7da5R215

Commits
-------

540817e5c Final reword
67460dc87 Reword and added the versionadded directive
3eec2ce07 Document file_relative Twig function

javiereguiluz committed:
Added the missing versionadded version

javiereguiluz committed:
minor #10038 [Finder] Add documentation and code on array as method parameter (jfredon)

This PR was merged into the master branch.

Discussion
----------

[Finder] Add documentation and code on array as method parameter

Documents symfony/symfony#27891

Commits
-------

361dde5f2 [Finder] Add documentation and code on array as method parameter

javiereguiluz committed:
Minor reword

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Added more links to screencast videos
  Explain another option to override bundle's entity mapping

colinodell committed:
Document the DivisibleBy constraint

thewilkybarkid committed:
Update translator fallback example to include ICU parents

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Remove useless $request
  Improve the docs about custom form theme config
  Use callable classes for custom Monolog processors
  Mentioned the audit_trail workflow option
  Problem with transactions
  Update custom_password_authenticator.rst
  Updated one screencast URL for Symfony 4
  Change dsn env var name
  Update dependency_injection.rst
  Update http_kernel_httpkernel_class.rst
  Remove example of register a service where its id match to its class

weaverryan committed:
Merge branch '4.1'

* 4.1:
  updating 2 old paths
  [WCM] Updating the Encore documentation + New recipe
  fix link: add missing colon after ":doc"
  Fix typo

weaverryan committed:
Merge branch '4.1'

* 4.1:
  Fixing bad syntax
  Fixed grammar issue in console.rst
  Specify version for symfony/skeleton
  [Process] document command-as-arrays

xabbuh committed:
[#10038] fix minor typos

xabbuh committed:
[#10038] minor versionadded wording tweaks

xabbuh committed:
add XML and PHP config examples

xabbuh committed:
minor #10155 add XML and PHP config examples (xabbuh)

This PR was merged into the master branch.

Discussion
----------

add XML and PHP config examples

Commits
-------

dc2bda4 add XML and PHP config examples

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Simplified the "Release Process" page
  [Workflow] Fix typo documentation
  Fix mismatched list items
  typos in sample code
  [#9988] fix minor typos
  [#10099] update XML and PHP config examples
  [#9969] fix a minor typo
  [#10022] fix a minor typo
  preUpdate Event Listener On Uploaded Imagery
  [#9757] fix rst syntax
  [#10124] fix XML attribute name
  [#10062] fix the code block
  [PHPUnitBridge] Explain how to show stack traces
  Fix docs on trusted hosts
  opcode optimizations

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Remove references to Silex
  Removing outdated note: AJAX requests are not set to target path
  Fixed some minor issues
  server fails to run on this tutorial
  fix XML attribute name
  fix cache option data type

weaverryan committed:
Merge branch '4.1'

* 4.1:
  added a space before "=" in code
  fix the markup
  Changing to absolute url, as requested
  Updated link to Event Listeners page
  Update form_customization.rst
  Revert "Fix typo on which YAML values must not be quoted"
  Fix typo (shard -> shared)
  Fix typo on which YAML values must not be quoted
  Change all occurrences of Controller to AbstractController
  fix `HandlerLocator`'s use statement

weaverryan committed:
Merge branch '4.1'

* 4.1:
  minor fixes
  Shorter array syntax for declaring numbers variable
  Twig is included with Symfony. No need to have the user run composer
  The `profiler-pack` is included in the composer package. It can be a bit confusing to the user with the extra stuff showing on the screen.
  Move the controller tip out of the annotation routes.

yceruto committed:
Documenting debug:twig command

weaverryan committed:
Merge branch '4.1'

* 4.1:
  {% if label is not same as(false) and required %}
  minor language tweak
  Update service_container.rst
  Fix wrong exception used in custom authentication provider example
  [Console] Replace useless usage of ContainerAwareCommand
  Shorten ternary operator
  Fix wrong usage in custom authenticator checking if password is invalid
  [Process] Fix typo in Process object initialization
  update hardcoded 5.1 version link
  Explain dump server usage outside of Symfony
  Add undocumented sortByAccessedTime, sortByChangedTime and sortByModifiedTime methods in finder component
  Update bugs.rst
  [#10200] remove trailing spaces
  action_method is missing $task variable
  [#10177] fix markup
  Clarifying when you need to or don't need to install the server bundle
  Fix bad link
  Update setup.rst
  Also requiring annotations so they can be used

xabbuh committed:
Merge branch '4.1'

* 4.1:
  remove acl
  Small fix to syntax
  Assume the user is using Flex when deploying

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Updated the doc link
  Update PHPUnit documentation link
  Move range to the comparison constraint types
  Update link to use latest docs
  Removed an unnecessary comment
  Added a tip about versioning when using Flex
  Use SCSS instead of CSS for code block
  Fix minor typo in the_big_picture.rst
  Removing calls to deprecated code
  [HttpFoundation] Small fixes on some examples
  Fixed Modifying Rendered Tables example
  Broken nginx XSendfile link

javiereguiluz committed:
Reworded and added the versionadded directive

javiereguiluz committed:
minor #10122 Update translator fallback example to include ICU parents (thewilkybarkid, javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

Update translator fallback example to include ICU parents

Changes the example to show https://github.com/symfony/symfony/pull/28070.

Commits
-------

7ccf60392 Reworded and added the versionadded directive
54631df93 Update translator fallback example to include ICU parents

javiereguiluz committed:
Minor reword

javiereguiluz committed:
minor #10108 [Routing] Add locale fallback to internationalized routes (Thomas Calvet, javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

[Routing] Add locale fallback to internationalized routes

Related PR : https://github.com/symfony/symfony/pull/27957

Commits
-------

02cc73977 Minor reword
c15df9d19 [Routing] Add locale fallback to internationalized routes

fmata committed:
Update dotenv.rst

javiereguiluz committed:
Reword

javiereguiluz committed:
minor #9568 Document the DotEnv::overload() method (fmata, javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

Document the DotEnv::overload() method

I'm not sure how to update the doc according my PR symfony/symfony#26859 which is pending review but feel free to comment if I did a mistake. Thank you.

Commits
-------

38e4f08c0 Reword
f8a3c0d34 Update dotenv.rst
63257596d Update dotenv.rst
c8ad2964c Update dotenv.rst

javiereguiluz committed:
Added docs for cookie_samesite option

javiereguiluz committed:
minor #10258 Added docs for cookie_samesite option (javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

Added docs for cookie_samesite option

This replaces #10202.

Commits
-------

4763b7b11 Added docs for cookie_samesite option

javiereguiluz committed:
Fixed the default value of cookie_samesite

javiereguiluz committed:
minor #10260 Fixed the default value of cookie_samesite (javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

Fixed the default value of cookie_samesite

Fixes the docs added in #10258.

Commits
-------

bb5ab5298 Fixed the default value of cookie_samesite

javiereguiluz committed:
minor #10232 Documenting debug:twig command (yceruto)

This PR was merged into the master branch.

Discussion
----------

Documenting debug:twig command

See https://github.com/symfony/symfony/pull/27981

Commits
-------

10260209a Documenting debug:twig command

javiereguiluz committed:
Minor reword

javiereguiluz committed:
minor #10121 [Validator] Document the DivisibleBy constraint (colinodell)

This PR was merged into the master branch.

Discussion
----------

[Validator] Document the DivisibleBy constraint

Documentation for the new ~`MultipleOf`~ `DivisibleBy` constraint proposed in https://github.com/symfony/symfony/pull/28069

Commits
-------

3270cca02 Document the DivisibleBy constraint

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  [Messenger] Error in XML configuration
  ! prefix referencing templates
  [Messenger] Fix syntax of Return Type Declaration
  Small fix to syntax of Return Type Declaration
  remove changelog toctree entry
  removed the changelog file
  removed Romain as part of the core team
  Update deployment.rst
  Fixed underline
  Adding check for environments
  Added pre-deployment tasks and removed Assetic deployment
  Added reference to deploying assets

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  stop using the deprecated @Method annotation

javiereguiluz committed:
minor #9875 Add Lock's PdoStore documentation (jderusse)

This PR was squashed before being merged into the master branch (closes #9875).

Discussion
----------

Add Lock's PdoStore documentation

Documenting https://github.com/symfony/symfony/pull/27456

Commits
-------

20511e80f Add Lock's PdoStore documentation

javiereguiluz committed:
Minor fixes

javiereguiluz committed:
Added docs for VAR_DUMPER_FORMAT

javiereguiluz committed:
Documented the "auto" mode of "cookie_secure"

ro0NL committed:
Remove \DateTimeInterface reference from constraint validators

chalasr committed:
[WCM][Console] Remove mentions about the deprecated ContainerAwareCommand class

javiereguiluz committed:
minor #10284 Documented the "auto" mode of "cookie_secure" (javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

Documented the "auto" mode of "cookie_secure"

This fixes #10283.

Commits
-------

af6265828 Documented the "auto" mode of "cookie_secure"

javiereguiluz committed:
minor #10275 Added docs for VAR_DUMPER_FORMAT (javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

Added docs for VAR_DUMPER_FORMAT

This fixes #10274.

Commits
-------

3924e1866 Added docs for VAR_DUMPER_FORMAT

javiereguiluz committed:
Added a missing reference to fix doc build

javiereguiluz committed:
Maintained a note about ContainerAwareCommand being deprecated

javiereguiluz committed:
minor #10307 [Console] Remove mentions about the deprecated ContainerAwareCommand (chalasr, javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

[Console] Remove mentions about the deprecated ContainerAwareCommand

Related to https://github.com/symfony/symfony/pull/28415

Commits
-------

4143835f9 Maintained a note about ContainerAwareCommand being deprecated
f3e8f0e4c [WCM][Console] Remove mentions about the deprecated ContainerAwareCommand class

javiereguiluz committed:
Added another missing reference to fix doc build

einenlum committed:
Add documentation about children method in DomCrawelr

dunglas committed:
[PropertyInfo] Inject the entity manager instead of the class metadata factory

dunglas committed:
[Serializer] Allow to pass a single value for the groups opt

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  [Serializer] Allow to specify a single value in @Groups
  [Validator] fix condition in if statement
  Fix broken link to Security tutorial
  PhpUnitBridge: Fix typo in note

dunglas committed:
[Serializer] Allow to access extra infos in name converters

dunglas committed:
Add a note block

dunglas committed:
[Serializer] Allow to access to the context and various other infos in callbacks and max depth handler

dunglas committed:
Fix, and document setCircularReferenceHandler's new parameters

dunglas committed:
[PropertyInfo] Add an extractor to guess if a property is initializable

dunglas committed:
RST

javiereguiluz committed:
Minor tweaks

javiereguiluz committed:
minor #10319 [PropertyInfo] Add an extractor to guess if a property is initializable (dunglas, javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

[PropertyInfo] Add an extractor to guess if a property is initializable

https://github.com/symfony/symfony/pull/26997
https://github.com/symfony/symfony/pull/24571
Closes #10272.

Commits
-------

8415f96cd Minor tweaks
eebca4a87 RST
f7ed14405 [PropertyInfo] Add an extractor to guess if a property is initializable

javiereguiluz committed:
Minor reword

javiereguiluz committed:
minor #10318 [Serializer] Allow to access to the context and various other infos in callbacks and max depth handler (dunglas, javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

[Serializer] Allow to access to the context and various other infos in callbacks and max depth handler

https://github.com/symfony/symfony/pull/27020
https://github.com/symfony/symfony/pull/27017

Commits
-------

672b37403 Minor reword
7c3e620ec Fix, and document setCircularReferenceHandler's new parameters
a4dad43a2 [Serializer] Allow to access to the context and various other infos in callbacks and max depth handler

javiereguiluz committed:
Minor reword

javiereguiluz committed:
minor #10317 [Serializer] Allow to access extra infos in name converters (dunglas, javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

[Serializer] Allow to access extra infos in name converters

https://github.com/symfony/symfony/pull/27021

Commits
-------

71afdca6d Minor reword
ae63f8a2b Add a note block
ca5e2e0ac [Serializer] Allow to access extra infos in name converters

javiereguiluz committed:
Replaced a "note" by a "tip"

javiereguiluz committed:
Added the missing versionadded directive

javiereguiluz committed:
minor #10312 [Serializer] Allow to pass a single value for the groups opt (dunglas, javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

[Serializer] Allow to pass a single value for the groups opt

https://github.com/symfony/symfony/pull/27503

Commits
-------

ae558f41d Added the missing versionadded directive
e4eae257d Replaced a "note" by a "tip"
5e0a91e53 [Serializer] Allow to pass a single value for the groups opt

javiereguiluz committed:
Added the missing versionadded directive

javiereguiluz committed:
minor #10311 [PropertyInfo] Inject the entity manager instead of the class metadata factory (dunglas, javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

[PropertyInfo] Inject the entity manager instead of the class metadata factory

symfony/symfony#27829

Commits
-------

44df1a71a Added the missing versionadded directive
1f87e0c4b [PropertyInfo] Inject the entity manager instead of the class metadata factory

javiereguiluz committed:
minor #10310 Add documentation about children method in DomCrawelr (Einenlum)

This PR was squashed before being merged into the master branch (closes #10310).

Discussion
----------

Add documentation about children method in DomCrawelr

Fixes #10288

Commits
-------

cbb057fed Add documentation about children method in DomCrawelr

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Update 3.3-di-changes.rst
  Fix typo and add weblink in core_team.rst
  use latest version in example
  Fix link for webpack encore
  Missing root node in yaml example
  Fix a typo in security/impersonating_user.rst
  [FrameworkBundle] Improve the DX of TemplateController

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Fixed Encore dir paths
  Change the public folder to web for 3.4
  Updated KnpUniversity links to SymfonyCasts
  correct a typo in a namespace loading
  There is no public folder in 3.4
  Fixed the link to Functional Tests
  Update templating.rst
  Update doctrine.rst
  Shorten Twig ternary expression
  [Serializer] Cache the normalizer to use when possible

Commit:
Document the port in access_control

yceruto committed:
Added support for multiple files or directories

yceruto committed:
adding missing "or"

yceruto committed:
Added support for multiple files or directories

yceruto committed:
typo

yceruto committed:
typo

javiereguiluz committed:
Added the versionadded directive

javiereguiluz committed:
minor #10368 [Translation] Added support for multiple files or directories in XliffLintCommand (yceruto, javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

[Translation] Added support for multiple files or directories in XliffLintCommand

See https://github.com/symfony/symfony/pull/28522

Commits
-------

e4494bda5 Added the versionadded directive
e68d2420b typo
56573038b Added support for multiple files or directories

javiereguiluz committed:
Added the versionadded directive

javiereguiluz committed:
minor #10367 [Yaml] Documenting support for multiple files or directories (yceruto, javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

[Yaml] Documenting support for multiple files or directories

See https://github.com/symfony/symfony/pull/28521

Commits
-------

23f64654b Added the versionadded directive
6cf8bef20 typo
a5546a685 adding missing "or"
1513f223d Added support for multiple files or directories

javiereguiluz committed:
minor #10043 Add documentation for using Zookeeper data store for lock component (Ganesh Chandrasekaran)

This PR was merged into the master branch.

Discussion
----------

Add documentation for using Zookeeper data store for lock component

This is a documentation update for new Feature of the Lock Component. This will be linked to the PR which will be created in https://github.com/symfony/symfony

Corresponding code PR: https://github.com/symfony/symfony/pull/27920

Commits
-------

2ba70f0d3 Add docs for Zookeeper Data Store.

javiereguiluz committed:
Minor tweaks

IndraGunawan committed:
Fix typo

javiereguiluz committed:
minor #10375 [Lock] Fix typo (IndraGunawan)

This PR was merged into the master branch.

Discussion
----------

[Lock] Fix typo

Commits
-------

79a29c882 Fix typo

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Minor typo fix on messenger xml config
  tweaked documentation about the profiler
  Change link to actual demo repo
  Remove unnecessary semicolons
  Change to short php echo tags
  [Encore] Make title in FAQ more clear
  [Forms] Update Bootstrap 4's documentation links to 4.1
  Mentioned how to redirect and maintain the query string
  Remove obsolete reference
  Minimalist default PSR-3 logger
  Fix #10250 - remove manual class discriminator configuration
  Use cross references for internal links
  Added a note on forms validation
  Mentioned the <optgroup> html tag explicitly
  Update redis_adapter.rst

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  how to handle AWS partial reverse proxy headers
  Update debug.rst

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Update name of the webpack encore package

javiereguiluz committed:
[Form] Deprecate searchAndRenderBlock returning empty string

javiereguiluz committed:
Deprecated not passing the root node name to config tree builders

javiereguiluz committed:
Documented the new date_label and time_label options

javiereguiluz committed:
Documented the binding of arguments by name and type

javiereguiluz committed:
Documented the setHeaderTitle() and setFooterTitle() methods

javiereguiluz committed:
Documented the setColumnMaxWidth() method

javiereguiluz committed:
minor #10388 Deprecated not passing the root node name to config tree builders (javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

Deprecated not passing the root node name to config tree builders

Fixes #10339.

Commits
-------

d70a0a6cf Deprecated not passing the root node name to config tree builders

javiereguiluz committed:
minor #10391 Documented the binding of arguments by name and type (javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

Documented the binding of arguments by name and type

Fixes #10023.

Commits
-------

eea8c4c77 Documented the binding of arguments by name and type

javiereguiluz committed:
minor #10394 Documented the setColumnMaxWidth() method (javiereguiluz)

This PR was merged into the master branch.

Discussion
----------

Documented the setColumnMaxWidth() method

This fixes #10030.

Commits
-------

00db5a3c3 Documented the setColumnMaxWidth() method

javiereguiluz committed:
minor #10392 Documented the setHeaderTitle() and setFooterTitle() methods (javiereguiluz)

This PR was squashed before being merged into the master branch (closes #10392).

Discussion
----------

Documented the setHeaderTitle() and setFooterTitle() methods

Fixes #10294.

Commits
-------

e29c5b138 Documented the setHeaderTitle() and setFooterTitle() methods

javiereguiluz committed:
minor #10387 [Form] Deprecate searchAndRenderBlock returning empty string (javiereguiluz)

This PR was squashed before being merged into the master branch (closes #10387).

Discussion
----------

[Form] Deprecate searchAndRenderBlock returning empty string

Fixes #10340.

Commits
-------

ac2ea07f6 [Form] Deprecate searchAndRenderBlock returning empty string

javiereguiluz committed:
minor #10389 Documented the new date_label and time_label options (javiereguiluz)

This PR was squashed before being merged into the master branch (closes #10389).

Discussion
----------

Documented the new date_label and time_label options

Fixes #10292.

Commits
-------

eee1eaa78 Documented the new date_label and time_label options

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  added mentoring and speaker mentoring sections
  Fix translation debug examples
  [Dotenv] Advise using the Composer --dev option
  Fixed some RST syntax issues
  Add note about configuration root node
  Add twig.runtime
  Updating language to remove ORM
  Recommend to install ORM pack instead of Doctrine bundle

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Fixed a missed reference
  Added a note about the usage of NotBlank

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Added a tip about atPath() method
  Remove useless use statements
  Added the missing class imports
  Update upload_file.rst for handling FileException
  Fix environment variable name

lexcast committed:
Update framework.rst

xabbuh committed:
Merge branch '4.1'

* 4.1:
  Fix typos in psr7.rst
  reword note about empty values during validation
  Update routing.rst
  Updated links

xabbuh committed:
Merge branch '4.1'

* 4.1:
  fix code example

javiereguiluz committed:
minor #10424 Mention vscode on IDE config (lexcast)

This PR was merged into the master branch.

Discussion
----------

Mention vscode on IDE config

<!--

If your pull request fixes a BUG, use the oldest maintained branch that contains
the bug (see https://symfony.com/roadmap for the list of maintained branches).

If your pull request documents a NEW FEATURE, use the same Symfony branch where
the feature was introduced (and `master` for features of unreleased versions).

-->
See symfony/symfony#28679

Commits
-------

0fdbdbbd1 Update framework.rst

dunglas committed:
[Serializer] Add an option to skip null values

javiereguiluz committed:
minor #10427 [Serializer] Add an option to skip null values (dunglas)

This PR was merged into the master branch.

Discussion
----------

[Serializer] Add an option to skip null values

https://github.com/symfony/symfony/pull/28661

Commits
-------

6349bd31e [Serializer] Add an option to skip null values

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Replace "set leave" with leave only
  Improved the explanation of naming in PHP and Twig

yceruto committed:
Documenting Options argument for closure deprecation func

hiddewie committed:
Added docs for LDAP user provider with default (null) values

hiddewie committed:
Add deprecation notice for --env and --no-debug

javiereguiluz committed:
minor #10302 Remove \DateTimeInterface reference from constraint validators (ro0NL)

This PR was squashed before being merged into the master branch (closes #10302).

Discussion
----------

Remove \DateTimeInterface reference from constraint validators

See https://github.com/symfony/symfony/pull/25015

Commits
-------

47b59177d Remove \DateTimeInterface reference from constraint validators

javiereguiluz committed:
Wrapped some long lines

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  typo fix in speaker mentoring
  Wrap the entity namespace in single quotes
  Improved the article about session flashes
  add example for cache pool configuration
  Added link to bcrypt (Wikipedia)
  Added a caution note about the checkMX option
  clarify the datetime format option
  Update configuration.rst
  Update doctrine.rst
  Added `charset` "query string"
  Resolve T_STRING
  You do not need to re-add the command

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  fix the command output
  Update search location of serializer configuration
  Fixed the logic of the custom password authenticator

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Removed the Action suffix in controller methods
  Update form_login_setup.rst
  [Form][Validation] User $user instead of $users
  fix server status and stop command descriptions
  Added 'Default' validation group
  Update Callback.rst
  [Encore] unify the name of app file

javiereguiluz committed:
Updated the form type extension article

javiereguiluz committed:
minor #10468 Updated the form type extension article (javiereguiluz)

This PR was squashed before being merged into the master branch (closes #10468).

Discussion
----------

Updated the form type extension article

Fixes #10466.

Commits
-------

21baaa5e1 Updated the form type extension article

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Fix and improve JSON login docs
  [Configuration] Change the configuration to match framework.yaml/php
  [Serializer] Updated the list of loaded encoders
  [VarDumper] Enhance the ServerDumper documentation

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  [VarDumper] Add missing use statements

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Fixed the ts-loader version
  Updated some config to follow the previous example
  Update routing.rst

javiereguiluz committed:
minor #10359 [Security] - Document the port in access_control (roukmoute)

This PR was merged into the master branch.

Discussion
----------

[Security] - Document the port in access_control

Commits
-------

b142f578c Document the port in access_control

javiereguiluz committed:
Minor reword

javiereguiluz committed:
minor #10439 [OptionsResolver] Documenting Options argument for closure deprecation func (yceruto)

This PR was merged into the master branch.

Discussion
----------

[OptionsResolver] Documenting Options argument for closure deprecation func

See https://github.com/symfony/symfony/pull/28738

Commits
-------

abb51892e Documenting Options argument for closure deprecation func

javiereguiluz committed:
minor #10445 Add deprecation notice for --env and --no-debug (hiddewie)

This PR was merged into the master branch.

Discussion
----------

Add deprecation notice for --env and --no-debug

Fix #10444.

- Add deprecation notice for `--env` and `--no-debug`.
- Update some command examples to use the `APP_ENV` & `APP_DEBUG` environment variables.

Commits
-------

77e352219 Add deprecation notice for --env and --no-debug

javiereguiluz committed:
Minor reword

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Improve is_granted documentation
  Fix phpunit command
  Replace extra ...
  [Messenger] Fix messenger.routing send_and_handle config
  Mentioned the 60 second timeout of processes
  fix #310430: DomCrawler component selectButton() docs update

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Updated the command to run the Docker image
  [Doctrine Bridge] document priority for doctrine.event_listener tag
  Improved the docs of the instanceof config
  Add details on how the ClockMock::register works
  Add Dockerfile
  [Messenger] Register a handler for a specific bus
  Fix function name in expression
  [Routing] Fix path ressource
  Add missing semicolon in doc

Nek- committed:
Add some lines of doc for waitUntil() (Process)

javiereguiluz committed:
minor #10492 Add some lines of doc for waitUntil() (Process) (Nek-)

This PR was merged into the master branch.

Discussion
----------

Add some lines of doc for waitUntil() (Process)

This adds some documentation for https://github.com/symfony/symfony/pull/27742

Commits
-------

2814fea88 Add some lines of doc for waitUntil() (Process)

javiereguiluz committed:
Minor reword

javiereguiluz committed:
minor #10443 Added docs for LDAP user provider with default (null) values (hiddewie)

This PR was squashed before being merged into the master branch (closes #10443).

Discussion
----------

Added docs for LDAP user provider with default (null) values

Fix #10336.

 - Add documentation for two fields of the LDAP user provider (UID key and filter).

Commits
-------

c19e78dce Added docs for LDAP user provider with default (null) values

javiereguiluz committed:
Minor reword

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Add autoconfiguration note for MessageHandlers
  PhpFilesAdapter not invalidating cache

weaverryan committed:
Merge branch '4.1'

* 4.1:
  removing old reference
  Fixing missing link
  Tweaks based on feedback
  Reword
  Teak
  Fix some bugs, added missing note, removed dead code and revamping code
  Some punctuation stuff
  Many changes thanks for GREAT feedback from various people
  Overhauling the security section
  Update conditions.rst
  Update conditions.rst
  Update conditions.rst
  Update conditions.rst

javiereguiluz commented:
there was some problem when creating this pull request. But don't worry, this has happened to all of us (and to me, several times 😊).

Sometimes this is just because you created the changes in some branch, but selected a different branch on GitHub when creating the pull request. If that's the case, this could be solved just by changing the branch in this pull request using GitHub's interface.

Otherwise, and if you don't want to deal with Git issues, it's OK to close this and create a new pull request. Thanks!

mpiot committed:
[FormType] Add help_attr documentation

javiereguiluz committed:
minor #10511 [FormType] Add help_attr documentation  (mpiot)

This PR was squashed before being merged into the master branch (closes #10511).

Discussion
----------

[FormType] Add help_attr documentation

Add documentation about the new feature symfony/symfony#27043

Commits
-------

f6162bc41 [FormType] Add help_attr documentation

javiereguiluz committed:
Minor reword

javiereguiluz committed:
Fixed a minor code issue

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Mentioned the new file upload exception classes
  Fixing bad logic, caused by merge originally
  Update docs now that sessions are enabled automatically
  Mentioned user_checkers in the main security config reference
  If multiple guard authenticators have different providers, link to the details on chaining providers together.

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  Removed an unnecessary (and outdated) reference
  fix spacing

javiereguiluz committed:
Merge branch '4.1'

* 4.1:
  fixed use statement

javiereguiluz commented:
I'm closing here but you can open a new pull request with your proposed changes. Thanks.
